providers/implementations/signature/eddsa_sig.c

```diff
@@ -607,6 +690,12 @@ const OSSL_DISPATCH ossl_ed25519_signature_functions[] = {
     { OSSL_FUNC_SIGNATURE_SET_CTX_PARAMS, (void (*)(void))eddsa_set_ctx_params },
     { OSSL_FUNC_SIGNATURE_SETTABLE_CTX_PARAMS,
       (void (*)(void))eddsa_settable_ctx_params },
+    { OSSL_FUNC_SIGNATURE_SET_SIGNATURE,
+      (void (*)(void))ed25519_verify_set_signature },
+    { OSSL_FUNC_SIGNATURE_DIGEST_VERIFY_UPDATE,
```

Comment:
I have hijacked the digest_verify_update and final here.. probably a better idea to have their own functions?